crypto/vrf/vrf.go

```diff
@@ -38,178 +34,109 @@ const (
 	ProofSize        = 32 + 32 + intermediateSize
 )
 
-var (
-	ErrGetPubKey = errors.New("[vrf] Couldn't get corresponding public-key from private-key")
-)
+// PrivateKey represents a Curve25519 private key.
+type PrivateKey [PrivateKeySize]byte
```

Comment:
Yes, thanks. It makes sense to keep independent from the actual implementation/key-size used.